### PR TITLE
[chore|SDE] Helm charts update and release tag

### PR DESCRIPTION
**Pull request by adityagajbhiye9**
## Description

-  helm chart update with app version .
-  helm lint workflow update.
-  release tag 0.1.4 and app version 2.3.5.

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: charts/simpledataexchanger/values.yaml [hunk 1]
```diff
@@ -28,7 +28,8 @@ sdepostgresql:
   auth:
     postgresPassword: ""
     username: "sdeuser"
-    password: ""
+    password: ""y
```
Comment (sachinargade123) [inline charts/simpledataexchanger/values.yaml]: Please remove 'y' character

**Review by sachinargade123 — APPROVED** [review]
LGTM

File: charts/simpledataexchanger/values.yaml (hunk 1 above)
Comment (adityagajbhiye9) [reply charts/simpledataexchanger/values.yaml]: typo removed.

**Comment by adityagajbhiye9** [comment]
@adkumar1 @almadigabor  please review and merge.

**Review by adkumar1 — APPROVED** [review]
Looks good

**Review by almadigabor — APPROVED** [review]

**Pull request merged**